src/main/java/fr/esgi/al5/tayarim/services/ReservationService.java

```diff
@@ -147,7 +148,13 @@
 
     String status;
     if (isAdmin) {
-      status = "payed";
+      if(dateDepart.isBefore(LocalDate.now()) || dateDepart.isEqual(LocalDate.now())){
+        status = "done";
+      }else if(dateArrivee.isBefore(LocalDate.now()) || dateArrivee.isEqual(LocalDate.now())){
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '{' is not preceded with whitespace.